--- efivar-30/src/Makefile.orig	2016-11-10 20:02:50.692925296 +0100
+++ efivar-30/src/Makefile	2016-11-10 20:04:37.653753960 +0100
@@ -84,6 +84,7 @@
 		ln -fs $(x).1.$(VERSION) $(DESTDIR)$(libdir)/$(x).1 ;\
 		ln -fs $(x).1.$(VERSION) $(DESTDIR)$(libdir)/$(x) ;\
 		)
+	$(foreach x, $(STATICLIBTARGETS), $(INSTALL) -m 644 $(x) $(DESTDIR)$(libdir);)
 	$(INSTALL) -d -m 755 $(DESTDIR)$(PCDIR)
 	$(foreach x, $(PCTARGETS), $(INSTALL) -m 644 $(x) $(DESTDIR)$(PCDIR) ;)
 	$(INSTALL) -d -m 755 $(DESTDIR)$(includedir)/efivar
