--- efivar-0.20/src/Makefile.orig	2015-06-02 20:32:44.000000000 +0200
+++ efivar-0.20/src/Makefile	2015-06-28 20:25:30.902448688 +0200
@@ -36,10 +36,15 @@
 
 libefivar.so.$(SONAME_VERSION) :: $(EFIVAR_OBJECTS)
 
+libefivar.so.$(SONAME_VERSION) : LIBS+=$(EFIVAR_LIBS)
+
 libefiboot.a :: $(EFIBOOT_OBJECTS)
 
-libefiboot.so.$(SONAME_VERSION) :: $(EFIBOOT_OBJECTS)
+libefiboot.so.$(SONAME_VERSION) :: $(EFIBOOT_OBJECTS) libefivar.so
 
+libefiboot.so.$(SONAME_VERSION) : LIBS+=efivar
+libefiboot.so.$(SONAME_VERSION) : LIBFLAGS+=-L.
+
 efivar : efivar.o libefivar.so
 	$(CCLD) $(ccldflags) -L. -lefivar -o $@ $^ \
 		-lpopt $(foreach lib,$(EFIVAR_LIBS),-l$(lib))
